terraform/install.sh

```diff
@@ -174,6 +174,26 @@ applyTerraform() {
   terraform apply -auto-approve -var="billing_account=${billing_acct}" -var="project_id=${project_id}" -var="bucket_name=${bucket_name}"
 }
 
+installMonitoring() {
+  external_ip="";
+  while [ -z $external_ip ]; do
+     external_ip=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'); 
+     [ -z "$external_ip" ] && sleep 10; 
+  done;
+
+  gcp_monitoring_path="https://console.cloud.google.com/monitoring?project=$project_id"
+  log "Please create a monitoring workspace for the project by clicking on the following link: $gcp_monitoring_path"
+  read -p "When you are done, please press enter to continue"
+
+  log "Initializing terraform state for monitoring examples"
+  cd ./monitoring
+  terraform init -lock=false
+
+  log "Apply Terraform automation for monitoring examples"
```

Comment:
Update to: "Creating dashboards and uptime checks for monitoring..."

From user's perspective it's not important that this is done with Terraform or other tool.

Reply:
Changed the earlier comment to reflect this :)